Sources/Common/Core/ScalarsToColors/index.js

```diff
@@ -137,7 +133,7 @@ function vtkScalarsToColors(publicAPI, model) {
   //----------------------------------------------------------------------------
   publicAPI.resetAnnotations = () => {
     model.annotationArray = [];
-    model.annotatedValueMap = [];
+    model.annotatedValueMap = new Map();
```

Comment:
model should only have variables that should be serializable. If you don't respect it, you will break the pattern `vtk(inst.getState())` unless you have special handling inside an overridden version of getState().
If you need a Map, can it be private?

model.* is really for the protected vars.

Reply:
Ok, I will back out this change, then.

An Object instead of an Array was considered, but modern linters require the equivalent of a Map `has` to be called as `Object.prototype.hasOwnProperty.call(myObj, prop)` -- pretty annoying.

Would it be possible at some point to serialize a Map as an Object?